.github/workflows/daily-build.yml

```diff
@@ -0,0 +1,43 @@
+name: Daily build
```

Comment:
Shall we include trivy scan for daily build as well?

Reply:
@keizer619 addressed the suggestions